hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/adapters/IncompatibleMutationAdapter.java

```diff
@@ -47,6 +50,8 @@
   private final Connection connection;
   private final Configuration conf;
   private final MetricsExporter metricsExporter;
+  // Maximum timestamp that hbase can send to bigtable in ms.
+  static final long HBASE_EFFECTIVE_MAX_TIMESTAMP = Long.MAX_VALUE / 1000L;
```

Comment:
nit: Rename it to Bigtable_Effective_max timestamp. This is a bigtable limitation and has nothing to do wiht HBase.